final_project/UITestFixture.cs

```diff
@@ -17,21 +56,77 @@ public async Task Setup()
                 Headless = false
```

Comment:
browser already initialized in OneTimeSetUp, so no need to duplicate here
var playwrightDriver = await Playwright.CreateAsync();
            browser = await playwrightDriver.Chromium.LaunchAsync